.changeset/strange-pets-relax.md

```diff
@@ -0,0 +1,6 @@
+---
+"vue-demo-store": patch
+---
+
+- Add smooth scrolling on the routing change (excluding product variants)
```

Comment:
```suggestion
Add smooth scrolling on the routing change (excluding product variants)
```

using `-` causes ugly changeset entry, like here:
![image](https://github.com/user-attachments/assets/2e8dc440-bd1d-419d-91cd-e82a0a8190b4)

Reply:
yes, pure string without `-` is correct :)

Reply:
Ahh :D @patzick made me realise that it was a diff `+` ... ups 🙃